src/main/java/com/iridium/iridiumskyblock/commands/JoinCommand.java

```diff
@@ -64,6 +65,12 @@ public boolean execute(CommandSender sender, String[] args) {
                         return false;
                     }
 
+                    // PR: added UserJoinEvent
```

Comment:
I don't really get the point of this comment and we haven't done that before. This would just become messy really quick. 
```suggestion
```